src/components/NotesPage/NotesPageTopBar.js

```diff
@@ -36,6 +34,17 @@ export const NotesPageTopBar = () => {
     setIsMenuOpen(false);
   };
 
+  const pageWidth = document.documentElement.scrollWidth;
+  const pageOrientation = window.orientation ? window.orientation : 0;
+  let anchorPositionMenu = { top: 58, left: 0 };
+  if (pageWidth < 600) {
+    if (pageOrientation) {
+      anchorPositionMenu = { top: 42, left: 0 };
+    } else {
+      anchorPositionMenu = { top: 50, left: 0 };
+    }
+  }
```

Comment:
Maybe a better solution would be to use `anchorEl` ? https://stackoverflow.com/a/52551100/1657101

And set `anchorOrigin={{ vertical: "bottom",`, so we will have the menu appearing always at the bottom of the hamburger menu, independent of the appbar height.

Reply:
The material-ui developers closed this ticket in October, and there is still no change on the site https://material-ui.com/components/menus/.
I parsed the example https://material-ui.com/components/menus/#customized-menus but didn't understand the importance of getContentAnchorEl = {null}
But it doesn't work without it.